projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -32,13 +33,23 @@ class Initializer {
 	 *
 	 * @var string
 	 */
-	const PACKAGE_VERSION = '4.2.1';
+	const PACKAGE_VERSION = '4.3.0-alpha';
 
 	/**
 	 * HTML container ID for the IDC screen on My Jetpack page.
 	 */
 	const IDC_CONTAINER_ID = 'my-jetpack-identity-crisis-container';
 
+	const JETPACK_PLUGIN_SLUGS = array(
+		'jetpack-backup',
+		'jetpack-boost',
+		'zero-bs-crm',
+		'jetpack',
+		'jetpack-protect',
+		'jetpack-social',
+		'jetpack-videopress',
```

Comment:
I think we're missing `jetpack-search` here

Reply:
aha, good catch, thank you!!